tango/common/from_params.py

```diff
@@ -801,12 +801,17 @@ def to_params(self) -> Params:
         """
 
         def replace_object_with_params(o: Any) -> Any:
+            # print("object: ", o)
             if isinstance(o, FromParams):
                 return o.to_params().as_dict(quiet=True)
             elif isinstance(o, (list, tuple, set)):
                 return [replace_object_with_params(i) for i in o]
             elif isinstance(o, dict):
-                return {key: replace_object_with_params(value) for key, value in o.items()}
+                return {
+                    key: replace_object_with_params(value)
+                    for key, value in o.items()
+                    if key != "_state"
```

Comment:
_state expects the model state to be present, which is not the case in flax. This was a workaround but I suppose we could move this out

Reply:
Ideally we should have this logic associated with the flax object in question, rather than here. But if that's not feasible without a lot of refactoring, let's add a comment about why we've added this if-case.

Reply:
Turns out transformer models don't need this check, so removed it for now. When we add in linen models, it will be useful to see how to do this within the model class